carla_integration/VehicleConfigParams.yaml

```diff
@@ -0,0 +1,52 @@
+# Defines the ros parameters which define the characteristics of this host vehicle configuration
+
+# Double: Host vehicle length
+# Units: Meters
+vehicle_length: 5.0
+
+# Double: Host vehicle width
+# Units: Meters
+vehicle_width: 2.5
+
+# Double: Host vehicle height
+# Units: Meters
+vehicle_height: 3.0
+
+# Double: Distance from front axel to rear axel
+# Units: Meters
+vehicle_wheel_base: 2.79
+
+# Double: Radius of the tires
+# Units: Meters
+vehicle_tire_radius: 0.39
+
+# Acceleration limit
+# Units: m/s^2
+vehicle_acceleration_limit: 3.0
+
+# Deceleration limit
+# Units: m/s^2
+vehicle_deceleration_limit: 6.0
+
+# Max curvature rate
+# Units: (m^-1) / s
+vehicle_max_curvature_rate: 0.75
+
+#  (radians of a full steering wheel rotation) / (radians of tires with the longitudinal axis under full steer)
+vehicle_steering_gear_ratio: 16.863
+
+# Maximum steering angle
+# Unit: deg
+vehicle_steer_lim_deg: 29.2
+
+# steering dynamics time constant 
+# Unit: s
+vehicle_model_steer_tau : 0.3
```

Comment:
All of the above vehicle parameters need to match whatever your ackerman controller expects and that component should be using these values at runtime.

Reply:
These parameters are used in ackermann controller. The values need to be discussed with PATH. Will update values later.